libraries/AP_HAL_ChibiOS/hwdef/CubeYellow/hwdef.dat

```diff
@@ -438,6 +438,7 @@ define HAL_BATT_CURR_PIN 3
 define HAL_BATT_VOLT_SCALE 10.1
 define HAL_BATT_CURR_SCALE 17.0
 define HAL_GPIO_PWM_VOLT_PIN 3
+define HAVE_DUAL_CDC 1
```

Comment:
I think HAL_HAVE_USB_DUAL would be a better define

Reply:
I can change it to HAL_HAVE_DUAL_USB_CDC, because we may have mass storage as second endpoint at some point as well.

Reply:
please do, thanks